.github/workflows/support-frontend-build.yml

```diff
@@ -0,0 +1,62 @@
+name: Build support-frontend
+
+on:
+  pull_request:
+    branches: [ main ]
+
+env:
+  GU_SUPPORT_WORKERS_LOAD_S3_CONFIG: false
+
+jobs:
+  support_frontend_build:
+    if: github.actor != 'scala-steward'
```

Comment:
Perhaps this should exclude any PR from a fork, not just scala-steward?  I'm not sure how to identify that at the moment though.
The workflow will fail if the PR is coming from a fork anyway because it won't have access to secrets but would be better if it just skipped the workflow.